<?xml version="1.0"?>
<!-- Launches frame dropper and serial comms for the server side -->
<launch>
  
  <arg name="in_camera" default="/front"/>
  <arg name="out_camera" default="/front_web"/>
  <arg name="in_camera2" default="/back"/>
  <arg name="out_camera2" default="/back_web"/>
  <arg name="scale" default="1"/>
  <arg name="frame_skip" default="0"/>
  <arg name="image_rate" default="10"/>
  <arg name="depth_rate" default="4"/>
  <arg name="jpeg_quality" default="30"/>
  
  <!-- Launch image dropper -->
  <node pkg="frame_dropper" type="image_splitter_node" name="is_web">
    <remap from="/in" to="$(arg in_camera)"/>
    <remap from="/out" to="$(arg out_camera)"/>
    <remap from="/in_2" to="$(arg in_camera2)"/>
    <remap from="/out_2" to="$(arg out_camera2)"/>
    <param name="scale" value="$(arg scale)"/>
    <param name="frame_skip" value="$(arg frame_skip)" />
    <param name="publish_depth" value="false"/>
  </node>

</launch>
